app/geant-exporter/geant-exporter.cc

```diff
@@ -0,0 +1,246 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file geant-exporter.cc
+//---------------------------------------------------------------------------//
+/*! \brief Geant4 particle definition and physics tables exporter
+ *
+ *  The code exports particle information and physics tables built by the
+ *  selected physics list and geometry. Output data is stored into ROOT files
+ *  and read by Celeritas' Geant Importer class.
+ */
+//---------------------------------------------------------------------------//
+
+// C++
+#include <string>
+#include <vector>
+
+// Project
+#include "src/DetectorConstruction.hh"
+#include "src/ActionInitialization.hh"
+#include "src/GeantPhysicsTableParser.hh"
+
+// Geant4
+#include "G4RunManager.hh"
+#include "G4UImanager.hh"
+//#include "FTFP_BERT.hh"
+#include "G4VModularPhysicsList.hh"
+#include "G4GenericPhysicsList.hh"
+#include "G4PhysicsTable.hh"
+#include "G4VEmProcess.hh"
+#include "G4VEnergyLossProcess.hh"
+
+// ROOT
+#include "TFile.h"
+#include "TDirectory.h"
+#include "TTree.h"
+#include "TBranch.h"
```

Comment:
Ok. I thought the common rule was to use angle brackets for C++ includes, and double quotes for any other include (either my project or a package). So that rule will be found pretty much in any code I written. I will keep an eye on that to update whatever is needed, and will move the include ordering.

Reply:
Oh, interesting. No, actually the `<>` and `""` specifically tell the compiler what "search paths" to prefer. Angle brackets are for "system" headers, where the compiler prefers things outside of the current directory first; and quotes are for local includes.

See https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html